tests/unit/moduleapi/list.tcl

```diff
@@ -70,37 +70,49 @@ start_server {tags {"modules"}} {
         # delete from start of list (index 0)
         r del l
         r rpush l x y z
-        r list.edit l dd
+        set reply [r list.edit l dd]
+        assert_equal [lindex $reply 3] 0
+        assert_equal [lindex $reply 5] {z}
```

Comment:
It's a bit hard to see what this means. How about assert the full $reply, something like this?

    assert_equal "num_edits 2 index 0" $reply

Reply:
if we want to match individual field in each line, we can also convert it into a dict on tcl side (or even use RESP3 map).
but maybe we can just match the entire string as Viktor suggested

Reply:
Doen with change to RESP3 Map.